app/views/users/_user.html.erb

```diff
@@ -1,6 +1,11 @@
 <li>
   Name: <%= user.name %>
-  <span class="profile-link">
-    <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+  <span class="profile-link mx-2">
+    <%= link_to 'See Profile',  user_path(user), class: 'button' %>
+    <%= link_to friendship?(user)[0], friendship?(user)[1], method: friendship?(user)[2], class: 'button mx-2'%>
+    <% if !inverse_friendship_with(user).nil? && inverse_friendship_with(user).status == false %>
+      <%= link_to 'Reject', friendship_path(inverse_friendship_with(user)), method: :delete, class: 'button mx-2'%>
```

Comment:
* [ ] There should be absolutely NO logic in views. >Views are only for displaying information that became available to it from its corresponding controller not generating it. **Resources**
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
* [Rails View Helpers Best Practices](https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4)
* [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright